project/Scriptit.scala

```diff
@@ -53,7 +53,18 @@ object Scriptit {
   }
 
   def scriptitParser(scriptitBase: File, script: String): Parser[Seq[String]] =
-    Defaults.distinctParser(scriptitTests(scriptitBase, script).toSet, raw = false)
+    distinctParser(scriptitTests(scriptitBase, script).toSet, raw = false)
+
+  def distinctParser(exs: Set[String], raw: Boolean): Parser[Seq[String]] =
```

Comment:
1. Added a comment.
2. I don't think we have to do anything else for now. Hopefully these usages are rare.